### ISSUE TITLE
Support for more operators

### ISSUE THREAD

**Issue by LoHhhha** [LoHhhha/pmos_nn#1]
**Version 0.1.a202501 Adaptation List**
- Tensor input: like torch.ones()/torch.zeros()/torch.randn()/torch.full()
- Concat/Cat/Concatenate
- Stack: dstack/hstack/vstack/stack
- Permute
- Unsqueeze
- Squeeze
- Embedding
- BatchNormXd
- GroupNorm
- LayerNorm
